functional_testing/zrobot_templates/framework/zos_utils/utils.py

```diff
@@ -38,3 +41,62 @@ def create_container(self, **kwargs):
 
     def create_vm(self, **kwargs):
         return self.handle_blueprint('vm.yaml', **kwargs)
+
+    def check_vnc(self, vnc_ip):
+        vnc = 'vncdotool -s %s' % vnc_ip
+        result,error = self.execute_shell_commands(cmd="%s type %s key enter" % (vnc, repr('ls')))       
+        return error 
```

Comment:
why are u returning error? is there any helpful info other than error?

Reply:
done